src/StyledProps/index.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  space,
+  SpaceProps,
+  color,
+  ColorProps,
+  typography,
+  TypographyProps,
+  layout,
+  LayoutProps,
+  boxShadow,
+  BoxShadowProps,
+  border,
+  BorderProps,
+  compose,
+  overflow,
+  OverflowProps,
+} from "styled-system";
+import { textOverflow, TextOverflowProps } from "./textOverflow";
+import { cursor, CursorProps } from "./cursor";
+import { transform, TransformProps } from "./transform";
+import { transition, TransitionProps } from "./transition";
+import { visibility, VisibilityProps } from "./visibility";
+
+export const handleStyledProps = compose(
```

Comment:
Nit pick: why are we calling this `handleStyledProps`. When I see `handle...` in a React context my mind thinks event handler right away. Should this be called something else like `getStyledProps`, `withStyledProps`, `addStyledProps` or anything else you think of?